Makefile

```diff
@@ -39,6 +41,13 @@ run-group: ve
 	ANSIBLE_LIMIT=${GROUP} make run
 	vagrant destroy -f ${HOSTS}
 
+run-machine: ve
+	vagrant up ${MACHINE}
+	vagrant status ${MACHINE}
+	vagrant ssh-config ${MACHINE} > ssh_config
+	ANSIBLE_LIMIT=${MACHINE} make run
+	vagrant destroy -f ${MACHINE}
```

Comment:
```suggestion
	vagrant up $(MACHINE)
	vagrant status $(MACHINE)
	vagrant ssh-config $(MACHINE) > ssh_config
	ANSIBLE_LIMIT=$(MACHINE) make run
	vagrant destroy -f $(MACHINE)
```

Reply:
I think it works in both cases

Reply:
I'll update them both, to be consistent 👍